### PR TITLE
adding tooltip to ribbons in condensed card view

### PR DESCRIPTION
**Pull request by SidelineCory24**

### CODE REVIEW

File: src/sass/_condensedCard.scss [hunk 1]
```diff
@@ -232,6 +212,25 @@ $condensed-card-data-padding: 15px 20px 9px;
...
+      .ribbon-cn {
+        .fa {
+        margin-right: .2em;
+        }
+      }
+      
+      .ribbon-htf {
+        .fa {
+        margin-right: .2em;
+        }
```
Comment (elizabeth-jimenez) [inline src/sass/_condensedCard.scss:231]: you can combine these styles, since they have the same values

File: src/sass/_condensedCard.scss [hunk 2]
```diff
@@ -232,6 +212,25 @@ $condensed-card-data-padding: 15px 20px 9px;
     flex-direction: column;
     height: 100%;
     position: relative;
+
+    .ribbon-container-condensed {
+      position: absolute;
+      right: -2px;
+      top: -1px;
+      z-index: 2;
```
Comment (elizabeth-jimenez) [inline src/sass/_condensedCard.scss:220]: 🤔 i have no idea if we can do this, but what if we try `$ribbon-condensed-card-z + 1`? That's probably not even a thing lol

Comment (SidelineCory24) [reply src/sass/_condensedCard.scss:220]: I typed this then deleted it. I'll try it out lol.

File: src/sass/_condensedCard.scss (hunk 1 above)
Comment (SidelineCory24) [reply src/sass/_condensedCard.scss:231]: done

File: src/sass/_condensedCard.scss (hunk 2 above)
Comment (SidelineCory24) [reply src/sass/_condensedCard.scss:220]: This is works

Comment (elizabeth-jimenez) [reply src/sass/_condensedCard.scss]: nice!

**Comment by SidelineCory24** [comment]
The CN, HTF, and SND ribbons are wrapped in StaticDev causing the tooltip to align different than the handshake ribbon tooltip.

File: src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx [hunk 3]
```diff
@@ -37,6 +38,55 @@ const ResultsCondensedCardTop = ({
...
   const link = `/${isProjectedVacancy ? 'vacancy' : 'details'}/${position.id}${isTandem ? '?tandem=true' : ''}`;
 
+  const ribbons = (
+    <div className="usa-grid-full post-ribbon-container">
+      <div className="ribbon-container-condensed">
+        {
+          hasHandshake &&
+          <Tooltip
+            title="Handshake"
+            offset={-30}
```
Comment (elizabeth-jimenez) [inline src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx:48]: adding `arrow` prop may be more consistent with our other tooltips

**Comment by elizabeth-jimenez** [comment]
> The CN, HTF, and SND ribbons are wrapped in StaticDev causing the tooltip to align different than the handshake ribbon tooltip.

Ah, nice observation!

**Comment by elizabeth-jimenez** [comment]
The position link is still activating when hovering on the ribbons for me

image

File: src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx (hunk 3 above)
Comment (SidelineCory24) [reply src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx:48]: done

File: src/sass/_condensedCard.scss [hunk 4]
```diff
@@ -128,16 +128,9 @@ $condensed-card-data-padding: 15px 20px 9px;
       position: absolute;
       right: -2px;
       top: -1px;
-      pointer-events: none;
```
Comment (elizabeth-jimenez) [inline src/sass/_condensedCard.scss:131]: `.ribbon-container` is now deprecated, so we can go ahead and delete that (line 120)

**Comment by SidelineCory24** [comment]
> The position link is still activating when hovering on the ribbons for me
> 
> image

`pointer-events: none;` is needed back.
*Note to self, don't remove Mike's fixes😆

File: src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx [hunk 5]
```diff
@@ -94,9 +122,12 @@ const ResultsCondensedCardTop = ({
   };
 
   return (
-    <Link to={link} {...containerProps} title="View details for this position">
-      {innerContent}
-    </Link>
+    <div {...containerProps}>
+      {ribbons}
+      <Link to={link} title="View details for this position">
```
Comment (elizabeth-jimenez) [inline src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx:127]: im not sure, but maybe keep `{...containerProps}` with the Link will fix the tooltip/link thing

Comment (elizabeth-jimenez) [reply src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx:127]: nope, but i'll look into it more

Comment (elizabeth-jimenez) [reply src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx:127]: `<div className="condensed-card-top">
      {ribbons}
      <Link to={link} {...containerProps} title="View details for this position">
        {innerContent}
      </Link>
    </div>`

File: src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx [hunk 6]
```diff
@@ -37,6 +38,55 @@ const ResultsCondensedCardTop = ({
 
   const link = `/${isProjectedVacancy ? 'vacancy' : 'details'}/${position.id}${isTandem ? '?tandem=true' : ''}`;
 
+  const ribbons = (
+    <div className="usa-grid-full post-ribbon-container">
```
Comment (elizabeth-jimenez) [inline src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx:42]: we may be able to remove `usa-grid-full`?

Comment (SidelineCory24) [reply src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx:42]: done

File: src/sass/_condensedCard.scss (hunk 4 above)
Comment (SidelineCory24) [reply src/sass/_condensedCard.scss:131]: done

File: src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx [hunk 7]
```diff
@@ -37,6 +38,59 @@ const ResultsCondensedCardTop = ({
...
+          >
+            <StaticDevContent>
+              <HardToFill showText={false} className="ribbon-condensed-card" />
+            </StaticDevContent>
+          </Tooltip>
+        }
+        {
+          isServiceNeedDifferential &&
+          <Tooltip
+            title="Serive need differential"
```
Comment (elizabeth-jimenez) [inline src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx:80]: 😅  typo

**Review by elizabeth-jimenez — APPROVED** [review]
Just the typo, but besides that 🍬 🍰 🍥 👍

File: src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx (hunk 7 above)
Comment (SidelineCory24) [reply src/Components/ResultsCondensedCardTop/ResultsCondensedCardTop.jsx:80]: done

**Pull request merged**